components/brave_shields/adblock/rs/Cargo.lock

```diff
@@ -0,0 +1,755 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```

Comment:
We shouldn't track local `Cargo.lock` files in version control. The browser build only looks at the one in `build/rust`.

Reply:
added by mistake, will remove

Reply:
will add `.gitignore` as well